homeassistant/components/squeezebox_player/media_player.py

```diff
@@ -178,16 +175,18 @@ async def async_turn_off(self) -> None:
         """Turn off device."""
         await self.player.power(False)
 
-    async def async_play_media(self, media_type: str, media_id: str, **kwargs) -> None:
+    async def async_play_media(
+        self, media_type: str | None, media_id: str, **kwargs
```

Comment:
The media_type parameter is required in the service call. Maybe add an intermediate local variable with another name that can be optional?